R/aqi_equation.R

```diff
@@ -0,0 +1,87 @@
+
+#' @title Calculate Air Quality Index (AQI) and classification
+#'
```

Comment:
Please add  description of the function after "#' @description" description tag.